tests/testapp/models.py

```diff
@@ -16,6 +16,23 @@
 db = databases.get("test")
 
 
+class DummyCache(object):
```

Comment:
maybe simple in-memory variant in sorcery itself? are there other libs there which can plug and play with this interface?

Reply:
cool. then prolly a good idea to add to somewhere